<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomChoose</class>
 <widget class="QWidget" name="RoomChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>443</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>808</width>
       <height>79</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>808</width>
       <height>78</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="HacListWidget" name="listWidget">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
       <property name="backgroundTransparent">
        <bool>true</bool>
       </property>
       <property name="notIntersetedItems">
        <bool>true</bool>
       </property>
       <property name="gridCount">
        <size>
         <width>5</width>
         <height>1</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Master Bedroom</string>
        </property>
        <property name="icon">
         <iconset resource="../res/hac01.qrc">
          <normaloff>:/hac01/images/room-normal.png</normaloff>
          <activeoff>:/hac01/images/room-active.png</activeoff>:/hac01/images/room-normal.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Secondary Bedroom</string>
        </property>
        <property name="icon">
         <iconset resource="../res/hac01.qrc">
          <normaloff>:/hac01/images/room-normal.png</normaloff>
          <activeoff>:/hac01/images/room-active.png</activeoff>:/hac01/images/room-normal.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Guest Bedroom</string>
        </property>
        <property name="icon">
         <iconset resource="../res/hac01.qrc">
          <normaloff>:/hac01/images/room-normal.png</normaloff>
          <activeoff>:/hac01/images/room-active.png</activeoff>:/hac01/images/room-normal.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Living Room</string>
        </property>
        <property name="icon">
         <iconset resource="../res/hac01.qrc">
          <normaloff>:/hac01/images/room-normal.png</normaloff>
          <activeoff>:/hac01/images/room-active.png</activeoff>:/hac01/images/room-normal.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dining Room</string>
        </property>
        <property name="icon">
         <iconset resource="../res/hac01.qrc">
          <normaloff>:/hac01/images/room-normal.png</normaloff>
          <activeoff>:/hac01/images/room-active.png</activeoff>:/hac01/images/room-normal.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Study Room</string>
        </property>
        <property name="icon">
         <iconset resource="../res/hac01.qrc">
          <normaloff>:/hac01/images/room-normal.png</normaloff>
          <activeoff>:/hac01/images/room-active.png</activeoff>:/hac01/images/room-normal.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Washing Room</string>
        </property>
        <property name="icon">
         <iconset resource="../res/hac01.qrc">
          <normaloff>:/hac01/images/room-normal.png</normaloff>
          <activeoff>:/hac01/images/room-active.png</activeoff>:/hac01/images/room-normal.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="prevButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../res/hac01.qrc">
         <normaloff>:/hac01/images/go-previous.png</normaloff>:/hac01/images/go-previous.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="nextButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../res/hac01.qrc">
         <normaloff>:/hac01/images/go-next.png</normaloff>:/hac01/images/go-next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HacListWidget</class>
   <extends>QListWidget</extends>
   <header>haclistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/hac01.qrc"/>
 </resources>
 <connections/>
</ui>
